### PR TITLE
#2011 fix Bring back the welcome post to the customer upon project cr…

### LINKED ISSUE #2011: [$40] Bring back the welcome post to the customer upon project creation
When the user lands on the dashboard after their new project is created, the system used to create a welcome post from Coder.

It is really important to maintain the welcome feel of the user experience, thus let's reinstate the initial post that drives the user to go and complete their specification, and explains what they should expect next.

### PR DESCRIPTION
**Pull request by jankyzhang**
fix #2011

### CODE REVIEW

File: src/api/messages.js [hunk 1]
```diff
@@ -121,7 +121,7 @@ export function getTopicsWithComments(reference, referenceId, tag, removeCoderBo
       if (removeCoderBotTopics) {
         //remove coderBot posts
         const rTopics = _.remove(topics, i =>
-          [DISCOURSE_BOT_USERID, CODER_BOT_USERID, TC_SYSTEM_USERID].indexOf(i.userId) > -1
+          i.title.indexOf('Your project has been created, and we') === -1 && [DISCOURSE_BOT_USERID, CODER_BOT_USERID, TC_SYSTEM_USERID].indexOf(i.userId) > -1
```
Comment (vikasrohit) [inline src/api/messages.js]: We should avoid doing string comparison. Rather, we should not hide the system messages now as we are only posting for project creation now. fyi @gondzo

Comment (jankyzhang) [reply src/api/messages.js]: @vikasrohit ,@gondzo How to change about this bug ?
Just make it not hide the system messages now ?
Please make a decision .
Thanks

Comment (jankyzhang) [reply src/api/messages.js]: @vikasrohit , the new commit is about the requirement '**not hide the system messages**'
Please check it.
Thanks

Comment (vikasrohit) [reply src/api/messages.js]: Thanks for the changes @jankyzhang . Please exclude the package-lock.json file from the PR as it should not have any changes for this fix. fyi @gondzo

Comment (jankyzhang) [reply src/api/messages.js]: @vikasrohit , thanks for the reply.
Sorry for the package-lock.json file.
I just submit a new commit .
Please check it.
Thanks.

**Pull request merged**